spa/src/analytics/index.ts

```diff
@@ -1,26 +1,34 @@
 import { AnalyticClient, ScreenNames } from "./types";
 import { useEffect } from "react";
 
-import { loadSoxAnalyticClient } from "./sox-analytics-client";
-import { noopAnalyticsClient } from "./noop-analytics-client";
+import { proxyAnalyticsClient } from "./proxy-analytics-client";
 
-const analyticsClient: AnalyticClient = loadSoxAnalyticClient() || noopAnalyticsClient;
+const analyticsClient: AnalyticClient = proxyAnalyticsClient();
```

Comment:
Should we separate concerns here? We're currently combining the creation of the analytics client instance with sending screen events.

Reply:
I refactor this code a little bit to just re-exporting the client. I think the intention of this class is to add as a facade to "analytics" component, that includes both the client and the screen event.